src/main/java/org/lantern/Cli.java

```diff
@@ -67,7 +67,7 @@ public Cli(String[] args) {
         }
         catch (final ParseException e) {
             printHelp(options, e.getMessage()+" args: "+Arrays.asList(args));
-            return;
+            System.exit(0);
```

Comment:
This was a drive-by fix.  I noticed that if I provided an invalid command-line argument, Lantern would start anyway.  That seemed dangerous/annoying.